package.json

```diff
@@ -22,15 +22,12 @@
   },
   "description": "Easily create a flexible REST interface for mongoose models",
   "devDependencies": {
-    "babel-preset-es2015": "^6.24.0",
-    "babel-register": "^6.26.0",
```

Comment:
We only used babel for istanbul, which I also removed. If we want coverage, we should probably switch to a more modern tool (maybe nyc?) but I'm not super familiar with this stuff.